src/config/config.json

```diff
@@ -0,0 +1,23 @@
+{
+  "development": {
+    "username": "root",
+    "password": null,
+    "database": "database_development",
+    "host": "127.0.0.1",
+    "dialect": "mysql"
```

Comment:
dialect should be `postgres`.

Reply:
Changed to postgre dialet while I was setting up the DB, you should see it fixed in mn/setdatabase.